### PR TITLE
Add sameAsOrigin in AllowedOrigins

### PR DESCRIPTION
**Pull request by makleso6**
make pr for #44 issue

### CODE REVIEW

**Review by mbarnach — APPROVED** [review]
Looks good to me!

File: Tests/KituraCORSTests/TestCORSOptions.swift [hunk 1]
```diff
@@ -207,6 +207,39 @@ class TestCORSOptions : XCTestCase {
             }, headers: ["Origin" : "http://api.bob.com"])
         }
     }
+    
+    func testSimpleWithSameAsOrigin() {
+        performServerTest(router) { expectation in
+            self.performRequest("get", path:"/same", callback: {response in
+                XCTAssertNotNil(response, "ERROR!!! ClientRequest response object was nil")
+                guard (response != nil) else {
```
Comment (dannys42) [inline Tests/KituraCORSTests/TestCORSOptions.swift]: Why not something like this?

```
guard let response = response else {
    XCTFail("Missing response")
    return
}
```

File: Tests/KituraCORSTests/TestCORSOptions.swift [hunk 2]
```diff
@@ -207,6 +207,39 @@ class TestCORSOptions : XCTestCase {
             }, headers: ["Origin" : "http://api.bob.com"])
         }
     }
+    
+    func testSimpleWithSameAsOrigin() {
+        performServerTest(router) { expectation in
```
Comment (dannys42) [inline Tests/KituraCORSTests/TestCORSOptions.swift:213]: FYI, I like to set variables at the top of my tests like this:

```
let inputValue = "http://api.bob.com"
let expectedValue = inputValue
var resolvedValue: String
```

In this case, when doing comparison values, such as:

```
XCTAssertEqual(resolvedValue, expectedValue, "reason")
```

It just gives a little more context as to what is expected of the test.

**Review by dannys42 — CHANGES_REQUESTED** [review]
Thanks for the PR!  Would you mind updating the README to include this new parameter as well?

**Comment by makleso6** [comment]
@dannys42 I made changes

**Review by dannys42 — APPROVED** [review]
Looks good, thanks!

**Comment by dannys42** [comment]
Looks like some unrelated errors running the unit tests.  I'm re-running them to see if the problem goes away.

**Comment by mbarnach** [comment]
Looks good now. We can merge! Thanks @makleso6 !

**Pull request merged**